hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/payload/TestAWSDmsAvroPayload.java

```diff
@@ -18,7 +18,6 @@
 
 package org.apache.hudi.payload;
 
-import org.apache.hudi.common.model.AWSDmsAvroPayload;
```

Comment:
can this test be moved to hudi-common? where the tested class belongs

Reply:
fixed